packages/dds/map/src/interfaces.ts

```diff
@@ -390,7 +398,9 @@ export interface ISharedMap extends ISharedObject<ISharedMapEvents>, Map<string,
  * If type is a value type then it must be amongst the types registered via registerValueType or we won't know how
  * to serialize/deserialize it (we rely on its factory via .load() and .store()).  Its value will be type-dependent.
  * If type is Shared, then the in-memory value will just be a reference to the SharedObject.  Its value will be a
- * channel ID. This type is legacy and deprecated.
+ * channel ID.
+ *
+ * @deprecated TODO: what is intended to be used instead?
```

Comment:
My memory is a bit fuzzy on how this area evolved.  The core idea though is that the in-memory representation of the values is not immediately stringifiable, primarily due to handles currently.  These interfaces represent some intermediate state that is immediately stringifiable.